core/src/main/scala/org/apache/gearpump/transport/netty/Context.scala

```diff
@@ -63,9 +63,9 @@ import org.apache.gearpump.transport.netty.Context._
   }
 
 
-  def bind(name: String, lookupActor : ActorLookupById): Int = {
+  def bind(name: String, lookupActor : ActorLookupById, deserializeFlag : Boolean = true): Int = {
     val taskDispatcher = system.settings.config.getString("gearpump.task-dispatcher")
```

Comment:
Is this flag an application-level configuration? If so you can move it to the user configuration.